scripts/config/local.json

```diff
@@ -213,25 +209,137 @@
         "0xf29ae508698bdef169b89834f76704c3b205aedf",
         "0xa258C4606Ca8206D8aA700cE2143D7db854D168c"
       ],
-      "filter": "0x6e0a5725dD4071e46356bD974E13F35DbF9ef367"
+      "filter": "0x927b167526bAbB9be047421db732C663a0b77B11"
     }
   },
   "uniswap": {
     "v2": {
       "factory": "0x5C69bEe701ef814a2B6a3EDD4B1652CB9cc5aA6f",
       "initCode": "0x96e8ac4277198ff8b6f785478aa9a39f403cb768dd02cbee326c3e7da348845f",
+      "router": "0x7a250d5630B4cF539739dF2C5dAcb4c659F2488D",
+      "paraswapUniV2Router": "0x86d3579b043585A97532514016dCF0C2d6C4b6a1",
```

Comment:
I would prefer to keep projects well separated. Can we have `paraswapUniV2Router` and all the forks configuration within the Paraswap config under a `uniswapV2Forks` property?